x-pack/plugin/sql/src/test/java/org/elasticsearch/xpack/sql/TestUtils.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package org.elasticsearch.xpack.sql;
+
+import org.elasticsearch.xpack.sql.proto.Mode;
+import org.elasticsearch.xpack.sql.proto.Protocol;
+import org.elasticsearch.xpack.sql.session.Configuration;
+
+import java.util.TimeZone;
+
+public class TestUtils {
```

Comment:
Why a different class to hold the default Configuration? And [here](https://github.com/elastic/elasticsearch/pull/36562/files#diff-b25bdd2cac33ae7bd8be0fb216960cccR48) a slightly different configuration is created? (one is using `TimeZone.getTimeZone("Z")` while the other is using "UTC")

Reply:
I think it's better like that if this static variable is used only for testing to not reside in the main class, but I also don't know the diff between `Z` and `UTC`.

Reply:
1. that configuration is only used for tests; we shouldn't use a implicit one (the fact that it was used in clean was incorrect since all clean actions were reported as part of PLAIN)
2. I'll look into aligning them - `UTC` and `Z` return slightly different ZoneId which create different `ZonedDateTime` which has issues being serialized - hence why I raised an issue of moving away from `TimeZone` to `ZoneId` to be consistent since converting between the two has issues.
`TimeZone.getID("Z").equals(TimeZone.getID("UTC")` but the same is not true for `ZoneId`.